### PR TITLE
Test against Rails 8.0.0

### PR DESCRIPTION
**Pull request by tagliala**
- [x] #8501
- [x] #8502
- [x] https://github.com/svenfuchs/rails-i18n/pull/1130
- [x] cucumber/cucumber-rails#590

### CODE REVIEW

File: lib/active_admin/router.rb [hunk 1]
```diff
@@ -65,10 +65,10 @@ def resource_routes(config)
 
     def page_routes(config)
       page = config.underscored_resource_name
-      router.get "/#{page}" => "#{page}#index"
+      router.get "/#{page}", to: "#{page}#index"
       config.page_actions.each do |action|
         Array.wrap(action.http_verb).each do |verb|
-          build_route(verb, "/#{page}/#{action.name}" => "#{page}##{action.name}")
+          build_route(verb, "/#{page}/#{action.name}", to: "#{page}##{action.name}")
```
Comment (javierjulio) [inline lib/active_admin/router.rb:71]: I prefer we do this change in a separate PR to close out the issue since this is not dependent on Rails 8 tests. This route signature is already long supported and thus is backwards compatible.

Comment (tagliala) [reply lib/active_admin/router.rb:71]: @javierjulio yes, I've committed here just for testing and to make sure that there aren't other deprecations, then I will backport to a separate PR

Comment (javierjulio) [reply lib/active_admin/router.rb:71]: @tagliala ok great, thank you.

**Comment by tagliala** [comment]
```diff
-    cucumber-rails (3.0.0)
-      capybara (>= 3.11, < 4)
-      cucumber (>= 5, < 10)
-      railties (>= 5.2, < 8)
+    cucumber-rails (1.4.0)
+      capybara (>= 1.1.2)
+      cucumber (>= 1.2.0)
+      nokogiri (>= 1.5.0)
+      rails (>= 3.0.0)
```

**Comment by tagliala** [comment]
Cucumber Rails seems to be the last blocker. There are no unexpected downgrades and CI is green. Is there an action to check for downgrades?

**Comment by javierjulio** [comment]
Ok thank you. I’m not sure if there is.

**Review by javierjulio — APPROVED** [review]
Thanks!
